feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.iteneum.notification.presentation
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.network.DataState
+import com.iteneum.notification.domain.models.NotificationDomain
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * [NotificationViewModel] is the view model what work with the NotificationListView, here is all the information will be consumed and passed to the view
+ *
+ * @author Usiel Filiberto Garcia Jimenez
+ */
+@HiltViewModel
+class NotificationViewModel @Inject constructor() : ViewModel(){
+
+    var dataToday: MutableList<NotificationDomain> = mutableListOf()
+        private set
+
+    var dataWeek: MutableList<NotificationDomain> = mutableListOf()
+        private set
+
+    init {
+        getInformation()
+    }
+
+    /**
+     * [getInformation] are a corrutines what are listening what be the status of the response when we call to the API, for example Success, Error, Loading
+     *
+     * @author Usiel Filiberto Garcia Jimenez
+     */
+    fun getInformation() = viewModelScope.launch {
+        /* HERE WE ARE GOING TO BE LISTENING THE DATA STATE FROM WE SERVICE */
+        when (response) {
+            is DataState.Success -> {
+                response.data.forEach{notification ->
```

Comment:
Here we can filter all the information in only one list

Reply:
Done